model/src/form/form-metadata/index.ts

```diff
@@ -55,7 +56,7 @@ export const formMetadataStateSchema = Joi.object<FormMetadataState>().keys({
  * Joi schema for `FormMetadata` interface
  * @see {@link FormMetadata}
  */
-export const formMetadataSchema = formMetadataCreateSchema.append({
+export const formMetadataSchema = formMetadataInputSchema.append({
```

Comment:
Think we're supposed to tell Joi that we've appended `FormMetadata` types?

Although it might only want the appended key/values, not the whole interface

```suggestion
export const formMetadataSchema = formMetadataInputSchema.append<FormMetadata>({
```

Reply:
That's neat.

Reply:
Fixed in 97b16a03e5760c938d8c46dfea0320a5d8d44820